spec/services/otp_rate_limiter_spec.rb

```diff
@@ -23,11 +23,13 @@
   end
 
   describe '#increment' do
-    it 'sets the otp_last_sent_at' do
-      now = Time.zone.now
+    it 'updates otp_last_sent_at' do
+      tracker = OtpRequestsTracker.find_or_create_with_phone(current_user.phone)
+      old_otp_last_sent_at = tracker.reload.otp_last_sent_at
       otp_rate_limiter.increment
+      new_otp_last_sent_at = tracker.reload.otp_last_sent_at
 
-      expect(rate_limited_phone.otp_last_sent_at.to_i).to eq(now.to_i)
+      expect(new_otp_last_sent_at > old_otp_last_sent_at).to eq true
```

Comment:
we could also do a floating point comparison?

```ruby
expect(rate_limited_phone.otp_last_sent_at.to_f).to be_within(1.0).of(now.to_f)
```

Or use a comparison matcher?

```ruby
expect(new_otp_last_sent_at).to be > old_otp_last_sent_at
```